common/devdoc/refcount_requirements.md

```diff
@@ -30,17 +31,17 @@ It wraps the structure that needs to be ref counted into another structure that
 #define DEFINE_REFCOUNT_TYPE(type) \
 ```
 
-**SRS_REFCOUNT_01_001: [** `DEFINE_REFCOUNT_TYPE` shall define the create/create_with_Extra_size/destroy functions for the type `type`. **]**
+**SRS_REFCOUNT_01_001: [** `DEFINE_REFCOUNT_TYPE` shall define the create/create_with_Extra_size/Create_Flex/destroy functions for the type `type`. **]**
```

Comment:
>create/create_with_Extra_size/Create_Flex/destroy [](http://example.com/codeflow?start=67&length=49)

Meh the create/create_with_Extra_size and destroy seem not to match what is in the code as casing :-)

Reply:
Casing corrected to match the code.